Translations update from Kodi Weblate

weblate opened the pull request:
Translations update from [Kodi Weblate](https://kodi.weblate.cloud) for [Kodi add-ons: videoplayer inputstream/inputstream.adaptive](https://kodi.weblate.cloud/projects/kodi-add-ons-videoplayer-inputstream/inputstream-adaptive/).



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widget/kodi-add-ons-videoplayer-inputstream/inputstream-adaptive/horizontal-auto.svg)

CastagnaIT commented:
"dont merge" as weblate still generate updates with wrong folders (e.g. en_GB) on his branch

weblate committed:
Added translation using Weblate (Ossetian (os_os))

Added translation using Weblate (Kannada (India) (kn_in))

Deleted translation using Weblate (Ossetian (os_os))

Deleted translation using Weblate (Kannada (India) (kn_in))

Deleted translation using Weblate (Occitan (France) (oc_fr))

Deleted translation using Weblate (Occidental (ie_ga))

Deleted translation using Weblate (Irish (ga_ie))

Deleted translation using Weblate (Filipino)

Deleted translation using Weblate (Irish)

Deleted translation using Weblate (Hebrew)

Deleted translation using Weblate (English (United Kingdom))

Translated using Weblate (Irish)

Currently translated at 100.0% (75 of 75 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 89.3% (67 of 75 strings)

Added translation using Weblate (Maori)

Added translation using Weblate (Hebrew)

Added translation using Weblate (Irish)

Added translation using Weblate (English (United Kingdom))

Added translation using Weblate (Filipino)

Added translation using Weblate (Irish (ga_ie))

Added translation using Weblate (Telugu (India) (te_in))

Added translation using Weblate (Occidental (ie_GA))

Added translation using Weblate (Slovenian (sl_si))

Added translation using Weblate (Malay (ms_my))

Added translation using Weblate (Macedonian (mk_mk))

Added translation using Weblate (Occitan (France) (oc_fr))

Translated using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Currently translated at 100.0% (75 of 75 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (75 of 75 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 86.6% (65 of 75 strings)

Co-authored-by: Aindriú Mac Giolla Eoin <aindriu80@gmail.com>
Co-authored-by: Alexey <signfinder@gmail.com>
Co-authored-by: Havok Dan <havokdan@yahoo.com.br>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kai Sommerfeld <ksooo@users.noreply.kodi.weblate.cloud>
Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>
Co-authored-by: SecularSteve <fairfull.playing@gmail.com>
Co-authored-by: rimasx <riks_12@hot.ee>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-videoplayer-inputstream/inputstream-adaptive/bs_ba/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-videoplayer-inputstream/inputstream-adaptive/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-videoplayer-inputstream/inputstream-adaptive/ga/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-videoplayer-inputstream/inputstream-adaptive/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-videoplayer-inputstream/inputstream-adaptive/ru_ru/
Translation: Kodi add-ons: videoplayer inputstream/inputstream.adaptive
